src/gov/nasa/worldwind/render/SurfaceText.java

```diff
@@ -153,6 +155,23 @@ public void setPosition(Position position)
         this.onShapeChanged();
     }
 
+    /** {@inheritDoc} */
+    public double getRotationAngle()
+    {
+        return this.rotationAngle;
+    }
+
+    /**
+     * {@inheritDoc}
+     * <p>
+     * The angle of text rotation from the true north (clockwise)
+     */
+    public void setRotationAngle(double rotationAngle)
+    {
+        this.rotationAngle = rotationAngle;
```

Comment:
If you look at `SurfaceQuad` as an example: if you change the type to `Angle` you could add a null-check as follows:

    if (heading == null)
    {
        String message = Logging.getMessage("nullValue.HeadingIsNull");
        Logging.logger().severe(message);
        throw new IllegalArgumentException(message);
    }